src/kirby/components/modal/embedded-modal.component.ts

```diff
@@ -0,0 +1,3 @@
+export interface EmbeddedModalComponent {
```

Comment:
I'm not really fond of this, I think that should be abstracted away in the ModalComponent itself. If we need to close the modal from inside the rendered component there could be other ways to achieve that, see e.g.: https://angular.io/api/common/NgComponentOutlet#a-simple-example using `ngComponentOutlet` + `injector`

Reply:
completely agree with you, this is now implemented